lib/galaxy/managers/workflows.py

```diff
@@ -544,19 +546,21 @@ def _workflow_to_dict_instance(self, trans, stored):
                 # Eventually, allow regular tool parameters to be inserted and modified at runtime.
                 # p = step.get_required_parameters()
         item['inputs'] = inputs
-        item['annotation'] = self.get_item_annotation_str( trans.sa_session, stored.user, stored )
+        item['annotation'] = self.get_item_annotation_str( sa_session, stored.user, stored )
         steps = {}
         for step in workflow.steps:
-            steps[step.id] = {'id': step.id,
-                              'type': step.type,
-                              'tool_id': step.tool_id,
-                              'tool_version': step.tool_version,
-                              'annotation': self.get_item_annotation_str( trans.sa_session, stored.user, step ),
-                              'tool_inputs': step.tool_inputs,
-                              'input_steps': {}}
+            step_type = step.type
+            step_dict = {'id': step.id,
+                         'type': step_type,
```

Comment:
The whole point of that commit is to clean up the diff for a larger feature addition downstream ([subworkflows](https://github.com/common-workflow-language/galaxy/commit/71cf09e8ff2bf640e2287ad0cd5426ae198f2dcc)) which does reuse the variable so I'd prefer to keep it this way, it isnt hurting anything.